packages/trie/CHANGELOG.md

```diff
@@ -6,6 +6,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 5.0.0-beta.3 - 2022-08-10
+
+Beta 3 release for the upcoming breaking release round on the [EthereumJS monorepo](https://github.com/ethereumjs/ethereumjs-monorepo) libraries, see the Beta 1 release notes for the main long change set description as well as the Beta 2 release notes for notes on some additional changes ([CHANGELOG](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/devp2p/CHANGELOG.md)).
+
+### Root Hash Persistance
+
+The trie library now comes with a new constructor option `persistRoot` which is now enabled by default (**breaking**) and which allows to persist state root updates along write operations directly in the DB and therefore omits the need to manually set to a new state root, see PR [#2071](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2071), thanks to @faustbrian for the contribution! ❤️
+
+To switch back to the old behavior you can pass in the option set to `false` on instantiation:
```

Comment:
```suggestion
To switch to the new behavior you can pass in the option set to `true` on instantiation:
```

Reply:
Unless I'm misunderstanding, I don't think this is correct, since the new behavior (`true`) is default and therefore passing `true` has no effect.

Reply:
Also the code below specifies `false`

Reply:
Actually I wasn't aware of the latest iteration: `https://github.com/ethereumjs/ethereumjs-monorepo/pull/2123`. We should then update the text as you mentioned @acolytec3, update the example below, and add PR 2123 to the PR links above.

Reply:
👍